cli/build.rs

```diff
@@ -26,10 +25,11 @@ fn main() -> std::io::Result<()> {
     // and most of the time it doesn't do anything.
     println!("cargo:rerun-if-changed=.schema/last_run.uuid");
     fs::create_dir_all(".schema").expect("failed creating the .schema directory");
-    write(".schema/last_run.uuid", Uuid::new_v4().to_string()).expect("Failed writing uuid file");
+    fs::write(".schema/last_run.uuid", Uuid::new_v4().to_string())
+        .expect("Failed writing uuid file");
 
     let schema_url = env::var("APOLLO_GPAPHQL_SCHEMA_URL")
-        .unwrap_or_else(|_| "https://engine-graphql.apollographql.com/api/schema".to_owned());
+        .unwrap_or_else(|_| String::from("https://engine-graphql.apollographql.com/api/schema"));
```

Comment:
FWIW also scatter `to_string()` on `&str` here and there, which literally calls `String::from`. Personally I don't love that the common way is so many characters, and that it puts the actual string (what the reader is probably interested in) later, but no strong objections or anything.

Can you elaborate on this preference?

Reply:
so this is not a pref so much as when considering the file as a whole, there was already a pattern of calling `String::from`

Reply:
i don't really have a personal pref, i kinda dig having the type information up front, don't really care about character count? but as stated, i chose `String::from` here because the doc already had a predominant pattern of using it

i do have a pref for `to_string` over `to_owned` ... preferring `to_owned` in truly generic situations, and `to_string` when we know we want a string